pkg/types/types.go

```diff
@@ -116,6 +116,9 @@ type Advisory struct {
 	FixedVersion    string `json:",omitempty"`
 	AffectedVersion string `json:",omitempty"` // Only for Arch Linux
 
+	// Advisory can contain different fixed versions for different architectures
+	FixedVersions FixedVersions `json:",omitempty"`
```

Comment:
`Entries` is better to get aligned with Red Hat. I'm not sure if it is a good name, though.
https://github.com/aquasecurity/trivy-db/blob/c76cb31f21102a3d51f22c09e46dc6110200ce82/pkg/vulnsrc/redhat-oval/types.go#L160-L162

Reply:
I thought about it and couldn't find good name.
But you are right. We use `Enties` in RedHat, let's also use `Entries` here.

Done in https://github.com/aquasecurity/trivy-db/pull/321/commits/14b1e69623e02e77ffd327040011f50da3997643